### DISCUSSION TITLE
Add a new countdown feature for multiplayer lobbies (Aimed for tournaments)

### DISCUSSION THREAD

**Discussion by Leinou3** [ppy/osu#23054] [Ideas] [answered]
![Suijidiscussion](https://user-images.githubusercontent.com/95429234/228516953-4d29af4e-f737-45a2-8266-4947a56e7706.PNG)

Today in Catch Suiji Tournament 3 we had a discussion about how hiding UI and such can make you miss the first notes during the starts of maps. Also players might be looking at their other screens (Mappool and such) when the match starts and when there is no unmappable beginning the map starts right away giving the players a heart attack.

This is why I propose an idea Benita had that there would be a 5 second countdown before each map starts to give people time to hide UI and scores and such (shift tab and tab). I would add a new Team Mode that is called something like Team Tournament in the Team Mode menu that would have the feature.
![multiplayermode](https://user-images.githubusercontent.com/95429234/228510507-3497c270-43cd-45e4-b2eb-8c45d7fc8c9e.PNG)

What do you think?
Also if this is added it could be customizable through skinning (sounds for 5,4,3,2,1 and maybe visuals)

**Comment by peppy** [comment, marked as answer]
have you tried lazer? i don't think your feedback applies.
